yaml/shared/src/main/scala/org/virtuslab/yaml/internal/load/parse/Event.scala

```diff
@@ -29,18 +30,25 @@ object Event:
       extends Document
 
   sealed trait Node extends Event
+
+  final case class Alias(id: String, pos: Option[Position] = None) extends Event
+
   final case class Scalar(
       value: String,
       style: ScalarStyle = ScalarStyle.Plain,
-      pos: Option[Position] = None
+      pos: Option[Position] = None,
+      anchor: Option[String] = None
   ) extends Node
 
-  sealed trait Sequence                                  extends Node
-  case class SequenceStart(pos: Option[Position] = None) extends Sequence
-  case class SequenceEnd(pos: Option[Position] = None)   extends Sequence
+  sealed trait Sequence extends Node
+  case class SequenceStart(pos: Option[Position] = None, anchor: Option[String] = None)
+      extends Sequence
+  case class SequenceEnd(pos: Option[Position] = None) extends Sequence
 
-  sealed trait Mapping                                      extends Node
-  case class MappingStart(pos: Option[Position] = None)     extends Mapping
-  case class MappingEnd(pos: Option[Position] = None)       extends Mapping
-  case class FlowMappingStart(pos: Option[Position] = None) extends Mapping
-  case class FlowMappingEnd(pos: Option[Position] = None)   extends Mapping
+  sealed trait Mapping extends Node
+  case class MappingStart(pos: Option[Position] = None, anchor: Option[String] = None)
```

Comment:
I think, we should add Context field, which contains for example `pos` or `anchor`

Reply:
I'll do this later, in another PR.